console/pom.xml

```diff
@@ -375,6 +375,11 @@
       <artifactId>ldaptive</artifactId>
       <version>1.0.3</version>
     </dependency>
+	<dependency>
+	  <groupId>com.vladmihalcea</groupId>
+	  <artifactId>hibernate-types-52</artifactId>
+	  <version>2.8.0</version>
+	</dependency>    
```

Comment:
Not a bloker but when staring the server, it display  a big message telling us that we should use hypersistence optimizer, maybe there is a way to disable this message.

Reply:
ok it's here : https://github.com/vladmihalcea/hibernate-types#setting-the-hibernatetypesprintbannerfalse-configuration-setting

Reply:
Nice catch.